main.go

```diff
@@ -20,27 +20,21 @@ limitations under the License.
 package main
 
 import (
-	"os"
 
 	// Import the snap plugin library
-	"github.com/intelsdi-x/snap/control/plugin"
+	"github.com/intelsdi-x/snap-plugin-lib-go/v1/plugin"
 
 	// Import our collector plugin implementation
 	"github.com/intelsdi-x/snap-plugin-collector-ceph/ceph"
 )
 
 // meta data about plugin
 const (
-	name       = "ceph"
-	version    = 5
-	pluginType = plugin.CollectorPluginType
+	name    = "ceph"
+	version = 6
 )
 
 // plugin bootstrap
 func main() {
-	plugin.Start(
-		plugin.NewPluginMeta(name, version, pluginType, []string{}, []string{plugin.SnapGOBContentType}, plugin.ConcurrencyCount(1)),
-		ceph.New(),
-		os.Args[1],
-	)
+	plugin.StartCollector(ceph.New(), name, version, plugin.RoutingStrategy(plugin.StickyRouter))
```

Comment:
Actually StickyRouter makes 1 plugin per task, so every task is sticked with specific plugin, it is mostly used for processors, but there are no detailed information about difference with ConcurrencyCount(1). RoutingStrategy is actually not needed for that plugin, so I'll revert the default behavior.